.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
I want to figure out how to move development of official starters to the Gatsby monorepo where we can handle enforcing coding style better.